specificatie/genereervariant/openapi.yaml

```diff
@@ -1985,8 +2006,8 @@ paths:
           minItems: 2
           type: array
           example:
-          - 196733.51
```

Comment:
De example is niet correct. Volgens de definitie moet het een nummer zijn. De example is een nummer in een string